com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: "/app/share/DefleMask/DefleMask"
+separate-locales: false
+tags:
+  - proprietary
+
+build-options:
+    prepend-pkg-config-path: "/usr/lib/i386-linux-gnu/pkgconfig"
+    ldflags: "-L/app/lib32"
+    prepend-path: "/usr/lib/sdk/toolchain-i386/bin"
+    env:
+        CC: "i686-unknown-linux-gnu-gcc"
+        CXX: "i686-unknown-linux-gnu-g++"
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=dri"
+  - "--allow=multiarch"
+
+sdk-extensions:
+  - 'org.freedesktop.Sdk.Compat.i386'
+  - 'org.freedesktop.Sdk.Extension.toolchain-i386'
+
+add-extensions:
+    'org.freedesktop.Platform.Compat.i386':
+        directory: "lib/i386-linux-gnu"
+        version: '19.08'
+
+    'org.freedesktop.Platform.GL32':
+        directory: "lib/i386-linux-gnu/GL"
+        version: '1.4'
+        versions: 19.08;1.4
+        subdirectories: true
+        no-autodownload: true
+        autodelete: false
+        add-ld-path: 'lib'
+        merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
+        download-if: active-gl-driver
+        enable-if: active-gl-driver
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/gtk2/gtk2.json
```

Comment:
As you don't need icons provided by shared module, copy only `gtk2` itself from there directly to your manifest.